vcs/rsync.go

```diff
@@ -0,0 +1,45 @@
+package vcs
+
+import (
+	"io/ioutil"
+	"os/exec"
+	"path/filepath"
+)
+
+func init() {
+	Register(newRsync, "rsync")
+}
+
+type RsyncDriver struct{}
+
+func newRsync(b []byte) (Driver, error) {
+	return &RsyncDriver{}, nil
+}
+
+func (g *RsyncDriver) HeadRev(dir string) (string, error) {
+	return "n/a", nil
```

Comment:
Thanks for the comment, unfortunately I'm not sure what to do about that. I think with rsync it actually makes sense to pull. I would then do a rsync --delete src dst. Which would update the state of the source in sync. Unfortunately for that I would need the url in te Pull call, since it is not saved in the data directory as with git or other vcses.

How should I proceed in updating this branch?